source/configure/authentication-configuration-settings.rst

```diff
@@ -2176,21 +2176,39 @@ Select OpenID Connect service provider
 
 *Available in legacy Enterprise Edition E20*
 
-Choose whether OpenID Connect can be used for account creation and login. Options include:
++-------------------------------------------------------------------------------+---------------------------------------------------------------+
+| Use this setting to enable OpenID Connect, with these options:                | - System Config path: **Authentication > OpenID Connect**     | 
+|                                                                               | - ``config.json`` setting: N/A                                |
+| - **Do not allow login via an OpenID provider**                               | - Environment variable: N/A                                   |
+| - **GitLab** `(settings) <#gitlab-openid-settings>`__                         |                                                               |
```

Comment:
I think we need a verb in front of "settings" as a clear call to action, as "see settings".